src/classes/BDI_MigrationMappingUtility.cls

```diff
@@ -892,8 +883,7 @@ public without sharing class BDI_MigrationMappingUtility {
         private List<String> splitHelpText(String helpText) {
             String symbol;
             Boolean hasSpecialChar = helpText.containsAny('$&+,:;=?@#|\'<>.-^*()%!_');
```

Comment:
Could we define the `specialCharacters` list at the top of the method and then perform a `String.join()` when defining the `hasSpecialChar` boolean so that we don't have to hard code the special characters in full twice?

Reply:
@dfuller114 @psadie-sfdo - I like the idea of defining this list once like Daniel suggested, but I'm seeing the characters being escaped differently between the two ("." vs "\\." for instance) - not sure how that would work exactly but only like the change if we are positive it doesn't cause weird bugs.